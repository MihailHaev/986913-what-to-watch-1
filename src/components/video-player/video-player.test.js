import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const mock = {
  film: {
    title: `Bohemian Rhapsody`,
    img: `img/bohemian-rhapsody.jpg`,
    src: ``,
  },
};

it(`Video Player correctly renders`, () => {
  const {film} = mock;
  const tree = renderer
    .create(<VideoPlayer
      film={film}
      isPlaying={false}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
