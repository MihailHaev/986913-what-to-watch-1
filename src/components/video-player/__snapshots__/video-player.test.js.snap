// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video Player correctly renders 1`] = `
<video
  height="175"
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
  src=""
  width="280"
/>
`;
